docs/source/techniques/SANS-Toml-v1.rst

```diff
@@ -412,49 +546,144 @@ DET/OVERLAP x y
         max = 0.24
         use_fit = true
 
+FIT/CENTRE t1 t2
+----------------
 
-FIT/CENTRE x y
----------------
+This command was used to specify a time window within which
+the 'prompt spike' could be found in *detector* spectra. This
+information was used to remove the spike by interpolating
+along the time-of-flight distribution. Also see
+FIT/MONITOR t1 t2.
 
-**Replacement**
+Times were specified in microseconds.
+
+# David this command used to be used on LOQ. Does the note on
```

Comment:
So this predates me, so I went on the implementation

`FIT /CENTRE` is [ignored by the old parser](https://github.com/mantidproject/mantid/blob/2dcb0847d0c0250501c677c88dc7beee995ba662/scripts/SANS/sans/user_file/user_file_parser.py#L2366)
whilst 
`FIT /MONITOR` actually [was parsed](https://github.com/mantidproject/mantid/blob/main/scripts/SANS/sans/user_file/user_file_parser.py#L1766) and was used in [prompt peak corrections](https://github.com/mantidproject/mantid/blob/2dcb0847d0c0250501c677c88dc7beee995ba662/scripts/SANS/sans/user_file/txt_parsers/ParsedDictConverter.py#L1083) and AFAIK can be used on anything (at least there are no checks where it's [used](https://github.com/mantidproject/mantid/blob/main/scripts/SANS/sans/algorithm_detail/calculate_sans_transmission.py#L293))

Reply:
Ok, I'll edit the file accordingly.